src/components/collapsible-motion/collapsible-motion.js

```diff
@@ -65,16 +66,14 @@ const useToggleAnimation = (isOpen, toggle, minHeight) => {
     ? { height: 'auto' }
     : { height: getMinHeight(minHeight), overflow: 'hidden' };
 
-  let animation = null;
-
   // if state has changed
   if (typeof prevIsOpen !== 'undefined' && prevIsOpen !== isOpen) {
```

Comment:
before if the component re-rendered without a toggle, animation was reset to null. Now the animation stays.